src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -6,7 +6,7 @@
     <RootNamespace>System.Net.Requests</RootNamespace>
     <AssemblyName>System.Net.Requests</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <!-- Disable binplacing for now since we need to use the netcore50aot version of this assembly -->
+    <!-- Disable binplacing for now since we need to use the uapaot version of this assembly -->
     <BinPlaceILCInputFolder>false</BinPlaceILCInputFolder>
```

Comment:
Does this property do anything? Looks like left-over from the razzle build system. Delete?